nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -377,42 +440,82 @@ define([
          * 1. if there's a tab showing, stop() it and detach it
          * 2. update the tabs state to be selected
          * @param {string} tab id of the tab to display
+         * @param {string} fileType id of the filetype we're swapping to
          */
-        function toggleTab(tab) {
+        function toggleTab(tab, fileType) {
+            // if we're toggling the currently selected tab off,
+            // then it should be turned off.
+            if (tab === state.tab.selected && tab !== null && !fileType) {
+                tab = null;
+            }
             state.tab.selected = tab;
+            stopWidget();
```

Comment:
This is another feature from the app cell that should be emulated for consistency. If the currently selected tab is clicked again, it gets shut off - it just removes that tab and leaves the cell closed.